Mathlib/Order/Closure.lean

```diff
@@ -262,6 +298,11 @@ def ofCompletePred (p : α → Prop) (hsinf : ∀ s, (∀ a ∈ s, p a) → p (s
     (fun a ↦ hsinf _ <| forall_range_iff.2 fun b ↦ b.2.2)
     (fun a b hab hb ↦ iInf_le_of_le ⟨b, hab, hb⟩ le_rfl)
 
+theorem sInf_IsClosed {c : ClosureOperator α} {S : Set α}
+    (H : ∀ x ∈ S, c.IsClosed x) : c.IsClosed (sInf S) :=
+  isClosed_iff_closure_le.mpr <| le_of_le_of_eq c.monotone.map_sInf_le <|
+    Eq.trans (biInf_congr (c.isClosed_iff.mp <| H . .)) sInf_eq_iInf.symm
```

Comment:
I think the standard dot for function arguments is the middle dot:
```suggestion
  isClosed_iff_closure_le.mpr <| le_of_le_of_eq c.monotone.map_sInf_le <|
    Eq.trans (biInf_congr (c.isClosed_iff.mp <| H · ·)) sInf_eq_iInf.symm
```
and you could use `le.le_of_eq` and `.trans` dot notations if you like.

Reply:
Gah, I keep forgetting about the middle dot. When not contributing to mathlib I just use a period for everything. Thanks for the heads up! Also for some reason I really dislike `.trans` over `Eq.trans`